core-bundle/src/Command/UserListCommand.php

```diff
@@ -133,10 +133,10 @@ static function (string $field) use ($user) {
         return $rows;
     }
 
-    private function formatJson(Collection $users, array $columns): array
+    private function formatJson(?Collection $users, array $columns): array
     {
         if ([] === $columns) {
-            return $users->fetchAll();
+            return $users ? $users->fetchAll() : [];
```

Comment:
There should be an early return at the beginning of the method, because line 144 might still trigger the error.

```php
private function formatJson(?Collection $users, array $columns): array
{
    if (!$user) {
        return [];
    }

    if ([] === $columns) {
        return $users->fetchAll();
    }

    // …
```